### DISCUSSION TITLE
Minimum Order Quantity in Offer Changed Notification

### DISCUSSION THREAD

**Discussion by asohler-be1source** [amzn/selling-partner-api-models#4203] [Ideas]
Can we get the Minimum Order Quantity added to the Offer Changed Notification when it's present on the offer? Without this information it makes it extremely difficult to price our items.

ASIN B00006IFCX can be used as an example of this scenario.  The lowest priced offer is way below everything else due to it requiring 6 quantity to be purchased. Trying to price your sale on a single unit compared to 6 units isn't really a fair comparison. It makes ASIN's look unprofitable from a seller standpoint, when in reality comparing against the single unit offers it would be a competitive item.

**Comment by jawheral** [comment]
Hi @asohler-be1source,

Did you check the [B2B_ANY_OFFER_CHANGED](https://developer-docs.amazon.com/sp-api/docs/notification-type-values#b2b_any_offer_changed) notification? This should include a the quantity discounts (quantityTier property) which is probably what you are looking for.

I hope this helps!

Best,
Jawher
Selling Partner API Solutions Architect

**Reply by asohler-be1source** [reply]
I can see it in the B2B notification, and it looks like it's handled correctly there. How would we use this data to handle the B2C Amazon pricing?

For example:
 - B2B Notification shows 1 Qty Tier Price 4.65. This is good as we know how to accurately price against competitors.
 - B2C Notification shows Lowest Price 3.03. This doesn't help us since there isn't any way to tell that the customer has to buy 6 to get this price. They can't buy less than 6 from this seller.

Only way to get that B2B Qty information in the B2C notification right now would be to compare the notifications when pricing the B2C item. Is that the recommended best practice right now? If so is there any plans to highlight the Minimum Qty restrictions when they are present on the B2C notifications as well?

**Reply by jawheral** [reply]
Hi @asohler-be1source,

If you need to price based on Business prices as well, the B2B Any Offer Changed notification might be more suitable. You can always call the [Product Pricing API](https://developer-docs.amazon.com/sp-api/docs/product-pricing-api-v0-reference) to get more data and granularity based for example on OfferType (B2C or B2B), after you receive the notification.

I will need to check internally whether it is expected to have B2B prices as part of `LowestPrices`. I will update if I have any information. For now, please use the notifications in conjunction with the API to achieve your intended use case.

Best,
Jawher
Selling Partner API Solutions Architect

**Comment by asohler-be1source** [comment]
Hi jawheral,

We'll try to figure out how to use them in conjunction. But I just want to clarify since the earlier language may of been a bit confusing, it's the below feature. Looks like based on the documentation it should be only a B2B feature, but it's showing up on the B2C pages as well.

![image](https://github.com/user-attachments/assets/52ef2ec5-07a4-4321-9c8e-2919f5716703)

https://sellercentral.amazon.com/help/hub/reference/GF2W6LRWA5V5ES7Z?ref_=xx_GF2W6LRWA5V5ES7Z_a_r24_cont_sgsearch

**Reply by jawheral** [reply]
Hi @asohler-be1source,

Yes, while the MOQ is documented as business only, it seems that offers with MOQ are being showed and enforced to non-business customers. Do you see this issue with other ASINs as well or only the one you shared? if there are more, kindly share them.

Nevertheless, this explains why the ANY_OFFER_CHANGED notification is not including these offers. I will follow up on this with our internal teams and will update you.

Thanks,
Jawher
Selling Partner API Solutions Architect

**Reply by asohler-be1source** [reply]
Hi @jawheral. I think thats the only ASIN that we share with the competitors who are using that MOQ feature. But some other ASIN's are below that show the MOQ on the B2C site. I would assume the ANY_OFFER_CHANGED notification for those ASIN's has the same problem as well.

B002OTHOPE
B00006IBQJ
B000VM2O5Q
B004ETY8T2
B07SQFHLNG
B001FME1BQ
B0033F1WYW
